### PR TITLE
feat: keyboard shortcut for adding links

### LINKED ISSUE #1466: 🚀 - Keyboard shortcut for adding links
### Description

It would be nice if a link could be added using `Ctrl + K` keyboard shortcut.

### PR DESCRIPTION
**Pull request by hakimio**
Fixes #1466

As an added bonus, automatically selects word at the current text cursor position. So, you don't have to manually select the word you want to add a link to.

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Comment by hakimio** [comment]
Wait before merging.

File: projects/editor/extensions/link/index.ts [hunk 1]
```diff
@@ -1,8 +1,51 @@
...
+        const parent = $anchor.parent;
+        const textBefore = parent.textBetween(0, pos - start, undefined, '\uFFFC');
+        const textAfter = parent.textBetween(
+            pos - start,
+            parent.content.size,
+            undefined,
+            '\uFFFC',
+        );
+
+        const wordBefore = textBefore.replaceAll('￼', '').split(/\b/).pop();
```
Comment (hakimio) [inline projects/editor/extensions/link/index.ts:31]: @splincode Is there a way to force ESLint to stop replacing `/\ufffc/g` with `'￼'`?

Comment (splincode) [reply projects/editor/extensions/link/index.ts:31]: add eslint ignore for that line

Comment (hakimio) [reply projects/editor/extensions/link/index.ts:31]: Which rule?

Comment (splincode) [reply projects/editor/extensions/link/index.ts:31]: I don't know who is it


check by

```
$ npx eslint .
```

**Review by splincode — APPROVED** [review]

**Pull request merged**